src/plugins/vis_type_table/public/index.ts

```diff
@@ -9,3 +9,6 @@ import { TableVisPlugin as Plugin } from './plugin';
 export function plugin(initializerContext: PluginInitializerContext) {
   return new Plugin(initializerContext);
 }
+
+/* Public Types */
+export { TableVisExpressionFunctionDefinition } from './table_vis_fn';
```

Comment:
Why? Appears unused in the rest of this PR.

Reply:
Sorry I slipped this in because there is a TS error related to this in the table agg fn which references it. This fixes that TS issue

Reply:
Thank you!